.cirrus.yml

```diff
@@ -265,6 +265,26 @@ task:
       script:
         - ./dev/bots/firebase_testlab.sh
 
+    - name: web_smoke_test
```

Comment:
Should we reuse the `web_integration_tests` shard?

Reply:
It is very different as the content goes. Especially with the template. May be I am missing something. What is the advantage if we use it? We should also change/remove the template from that shard then.

Reply:
A smoke test is an integration test. It's just a simple integration test. So I thought it belonged in the integration test shard. We also probably don't want too many shards as they all run simultaneously and occupy machines. I imagine eventually we'll have a hundred integration tests, and we'll shard them into N shards as we see fit, similarly to what we do with widget tests. This way we don't have hand-maintained specialized shards.